### DISCUSSION TITLE
Screencasts video size max-width: 900 is too limited

### DISCUSSION THREAD

**Discussion by horaceho** [livewire/livewire#5170] [1. Help] [answered]
Please resize screencast to better fit browser frame. Before:

![Screenshot 2022-08-09 at 12 06 59](https://user-images.githubusercontent.com/6657/183562297-cb3f22d6-459f-4fa5-8f26-3cfb4a3da05d.png)

After:

![Screenshot 2022-08-09 at 12 07 45](https://user-images.githubusercontent.com/6657/183562316-e29f136d-cefd-42b1-997c-7f522fbd3c75.png)

**Comment by joshhanley** [comment, marked as answer]
@horaceho thanks for the suggestion! This has been changed 🙂
